recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - setuptools
   run:
     - protobuf >=3.12.0
```

Comment:
This `protobuf` requirement does not match what's stated by upstream.

Reply:
> This `protobuf` requirement does not match what's stated by upstream.

The `protobuf` requirement was modified to reflect the upstream requirements:

https://github.com/AnacondaRecipes/proto-plus-feedstock/blob/update_to_1.19.9/recipe/meta.yaml#L25

Here are the upstream requirements:

https://github.com/googleapis/proto-plus-python/blob/v1.19.9/setup.py#L39